CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/KDAB/cxx-qt/compare/v0.7.0...HEAD)
 
+### Added
+
+- Support for further types: `qreal`, `qint64`, `qintptr`, `qsizetype`, `quint64`, `quintptr`
```

Comment:
do we also want `qlonglong`, `qulonglong`, or `qptrdiff` ?

See https://doc.qt.io/qt-6/qttypes.html#types for full types in Qt

Reply:
Let's just go with what we need at the moment and add new types as needed.